web/cobrands/oxfordshire/base.scss

```diff
@@ -213,6 +213,24 @@ dd, p {
   background-color: mix(#fff, $primary, 85%);
 }
 
+// We want OCCLSS buttons to look like FMS buttons.
+.occlss-button--primary {
+    @include button-size();
+}
+
+// No border and slightly different padding for *all* types of button.
+.btn,
+#front-main #postcodeForm div input#sub,
+.green-btn,
+button.green-btn,
```

Comment:
There aren't any button with class `green-btn`. Also I don't think there are any input.green-btn (there are inputs, but nothing that would need the specificity?)

Reply:
Yup. I have a feeling these selectors have all just been cargo culted (in the words of hart.scss) from older cobrands. Maybe the extra specificity was required a while back, but I can’t spot anywhere that needs it now. 👍